manager/allocator/network.go

```diff
@@ -560,6 +533,59 @@ func (a *Allocator) deallocateNode(node *api.Node) error {
 	return nil
 }
 
+// allocateNetworks allocates networks in the store so far before we process
```

Comment:
> store so far before we process watched events

Sorry, not sure what you mean by this. 

Also, please add a comment for how the existingOnly flag is used.